packages/react/package.json

```diff
@@ -128,7 +128,7 @@
     "@jupyterlab/translation-extension": "4.0.3",
     "@jupyterlab/ui-components-extension": "4.0.3",
     "@lumino/default-theme": "2.1.2",
-    "@primer/react": "0.0.0-20230912151940",
+    "@primer/react": "36.4.0",
```

Comment:
@fcollonval  This introduces a failure Attempted import error: 'UnderlineNav' is not exported from '@primer/react/drafts' (imported as 'UnderlineNav').

I faced that some time ago. primer has a policy, maybe possible to use something more stable than 0.0.0-..., but for now, keeping 0.0.0-20230912151940 would be ok

Reply:
In which package did you see the failure - the components is not draft anymore so the import should be updated to `@primer/react`

Reply:
I have seen that failure here https://github.com/datalayer/jupyter-ui/actions/runs/7291010284/job/19869056711#step:7:1471